static/src/javascripts/projects/common/views/commercial/creatives/fabric.html

```diff
@@ -0,0 +1,17 @@
+<div
+    class="fabric"
+    style="
+        background-image: url(<%=backgroundImage%>);
+        background-color: <%=backgroundColour%>;
+        background-repeat: <%=backgroundRepeat%>;
+        background-position: <%=backgroundPosition%>;
+        background-attachment: <%=(backgroundScroll === 'reveal' ? 'fixed' : 'scroll')%>;
```

Comment:
Beware, I've had bad experiences with expressions in lodash template once they've been included in the optimized build. The `text` plugin is escaping some characters where it shouldn't... you might want to offset that computation in your module.